.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
```

Comment:
Copied from upstream, only change is adding our `main` branch to be `gocardless`

Reply:
this i'm assuming will be removed when we move the code back to upstream i guess?

Reply:
(the gocardless bit i mean 🤣 )